src/test/java/com/hotel/api/book/controller/HotelControllerTest.java

```diff
@@ -30,10 +34,23 @@ public void testGetRateRuleResponse() throws Exception {
         request.setCheckOutDate("02-01-2022");
         request.setRoomCount(1);
         request.setLanguageCode("ENG");
-        hotelRateRuleController.getRateRuleResponse(request);
-        verify(hotelRateRuleService,atLeast(1)).service(request);
-
+        hotelController.getRateRuleResponse(request);
+        verify(hotelRateRuleService, atLeast(1)).service(request);
+    }
 
+    @Test
+    public void testGetBookResponse() throws Exception{
+        BookRequest request = new BookRequest();
+        request.setLanguageCode("ENG");
+        request.setHotelCode("ab");
+        request.setCountryCode("in");
+        request.setRatePlanId("12ab");
+        request.setNumberOfRooms(1);
+        request.setGuestCount(1);
+        request.setStartDate("01-01-2022");
+        request.setEndDate("02-01-2022");
+        hotelController.getBookResponse(request);
+        verify(hotelBookService, atLeast(1)).service(request);
```

Comment:
1. why verify is written without when condition? where is when condition?
2. Already Book Request object created in BookRequestBuilderTest please reuse it here